src/main/java/core/basesyntax/bookstore/repository/user/RoleRepository.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.bookstore.repository.user;
+
+import core.basesyntax.bookstore.model.Role;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface RoleRepository extends JpaRepository<Role, Long> {
+    Role getFirstByRole(Role.RoleName role);
```

Comment:
better return Optional 
```suggestion
    Optional<Role> findByName(Role.RoleName role);
```